tests/conftest.py

```diff
@@ -1578,8 +1582,29 @@ def collect_db_dump(request, duthosts):
     yield
     collect_db_dump_on_duts(request, duthosts)
 
-@pytest.fixture(scope="module", autouse=True)
-def verify_new_core_dumps(duthost):
+def compare_config(running_config, config_db_bak):
+    different_config = {}
+    for key, value in config_db_bak.items():
+        if running_config[key] != value:
+            different_config.update({key: value})
+
+    if different_config:
+        logger.info("The difference between running config and bak config {}".format(different_config))
+        return True
```

Comment:
It makes more sense to return True if compare passed and return False if compare failed.

Reply:
done